common/changes/office-ui-fabric-react/malozano-ComboBoxDisableStyle_2017-10-11-23-28.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "\"Make label text gray when combobox is disabled\" ",
```

Comment:
Minor note; when you add the change comments, make them in this format:

ComponentName: thing that changed.

Reply:
Thanks for the feedback! Will do.